### PR TITLE
Rename application forms

### PR DESCRIPTION
**Pull request by novotnyjakub**
This is a cleanup PR to remove the `applikation` model namespace and place the forms under more concise namespace.

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 7fc7f1e041ed56e6acae0fc04123d2aab6db1ad2
  * Your commit message body contains a line that is longer than 72 characters

Guidelines are available at https://github.com/ministryofjustice/fr-staffapp

---
This message was auto-generated by https://gitcop.com

File: app/models/form_object.rb [hunk 1]
```diff
@@ -31,6 +31,10 @@ def save
     end
   end
 
+  def i18n_scope
+    "activemodel.attributes.#{self.class.name.underscore}".to_sym
```
Comment (colinbruce) [inline app/models/form_object.rb]: Nice!

**Comment by colinbruce** [comment]
This looks great!  Merge when green. :+1:

**Comment by dotemacs** [comment]
@novotnyjakub When moving the files around did you use 'git mv file_name...rb' to preserve the history?

**Comment by novotnyjakub** [comment]
I didn't use `git mv`, but made sure Git recognises the moves and history is preserved. I just double checked.

**Pull request merged**